src/tests/methods.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { MiddlewareRegistry } from "../MiddlewareRegistry";
+import { NextApiRequest } from "next";
+
+describe('MiddlewareRegistry.methods', () => {
```

Comment:
Nit -

This is testing the `methods` property of `MiddlewareConfig`, not a property of the `MiddlewareRegistry` class itself.